components/Nav.js

```diff
@@ -18,6 +18,11 @@ const links = {
   Leaderboard: {
     href: '/leaderboard',
   },
+  Stories, Art, Toons: {
```

Comment:
This is an invalid name due to how javascript works. You are basically attempting to create a new link called "Toons", and add in 2 existing links called "stories" and "art". That is how the compiler sees it.